Source/SvgElement.cs

```diff
@@ -282,6 +282,16 @@ protected internal virtual SvgAttributeCollection Attributes
             }
         }
 
+        protected bool Inherited { get; set; } = true;
+
+        protected internal TAttributeType GetAttribute<TAttributeType>(string attributeName, bool inherit, TAttributeType defaultValue = default(TAttributeType))
+        {
+            if (inherit)

```

Comment:
Why not just use `Inherited` here instead of adding that extra parameter?

Reply:
Thank you for your review.

Some attributes are always `false` like SvgFragment, so add that extra parameter.

Reply:
Oh...
`svg` element can nest another `svg` element.

Reply:
Right. So with that in mind, I still think that this parameter is not needed - it is better in my opinion to make sure that the `Inherited` property is set correctly, and use that inside `GetAttribute`

Reply:
Please compare "id" attribute with "xml: space" attribute.

Reply:
Ok, thanks - I had some mind block, obviously. Sorry for the peckering...